platform-configs/novacustom-ns50mu.robot

```diff
@@ -17,306 +17,18 @@ ${IPXE_BOOT_ENTRY}=                                 iPXE Network boot
 ${AUTO_BOOT_TIME_OUT_DEFAULT_VALUE}=                ${EMPTY}
 ${LAPTOP_EC_SERIAL_WORKAROUND}=                     ${TRUE}
 
-# Platform flashing flags
```

Comment:
Is it necessary to remove 90% of this file?

Reply:
I see only indentation changes here. I assume this comment was for older patch version.